source/app/views/new_game.erb

```diff
@@ -0,0 +1,24 @@
+<h2>Welcome To Rock, Paper, Scissors! </h2>
+
+
+<form action= '/game' method='post'>
+  <input type="hidden" name="player_1_id" value= '1'>
+  <input type="hidden" name="player_2_id" value= '2'>
+
+  <div id='player_1_input'>
```

Comment:
See how the code for player 1 and player 2 is duplicated? why not iterate through the players and just create a div for each one of them